opm/input/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -131,30 +131,26 @@ struct SummaryConfigContext {
                                                                            {"FMWSET", FMWSET_keywords},
                                                                            {"GMWSET", GMWSET_keywords}};
 
-    /*
-      This is a hardcoded mapping between 3D field keywords,
-      e.g. 'PRESSURE' and 'SWAT' and summary keywords like 'RPR' and
-      'BPR'. The purpose of this mapping is to maintain an overview of
-      which 3D field keywords are needed by the Summary calculation
-      machinery, based on which summary keywords are requested. The
-      Summary calculations are implemented in the opm-output
-      repository.
-    */
+    // This is a hardcoded mapping between 3D field keywords,
+    // e.g. 'PRESSURE' and 'SWAT' and summary keywords like 'RPR' and
+    // 'BPR'. The purpose of this mapping is to maintain an overview of
+    // which 3D field keywords are needed by the Summary calculation
+    // machinery, based on which summary keywords are requested.
     const std::map<std::string , std::set<std::string>> required_fields =  {
-         {"PRESSURE", {"FPR" , "RPR" , "BPR"}},
-         {"RPV", {"FRPV" , "RRPV"}},
-         {"OIP"  , {"ROIP" , "FOIP" , "FOE"}},
+         {"PRESSURE", {"FPR" , "RPR*" , "BPR"}},
```

Comment:
maybe align table while at it.

Reply:
> maybe align table while at it.

Good idea.  I've pushed an update to that effect.